bliss/encoder.py

```diff
@@ -103,7 +103,8 @@ def __init__(
 
         num_channels = len(self.bands)
         num_features_per_band = self._get_num_features()
-        self.model = Backbone(cfg=arch_dict, ch=num_channels, n_imgs=num_features_per_band)
+        model = Backbone(cfg=arch_dict, ch=num_channels, n_imgs=num_features_per_band)
+        self.model = torch.compile(model)
         self.tiles_to_crop = tiles_to_crop
```

Comment:
How long does it take to compile? If it's more than a few seconds, it'd probably be good to turn compiling off by default (using our config files) to help with interactive runs.

Reply:
Yeah that sounds like a good idea. Compilation takes ~50 seconds, so it'll be amortized in a full training run but not for simple tests.